Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM helioncf/ubuntu-core
+RUN apt-get update && apt-get install -y \
+  nodejs \
+  git-core curl zlib1g-dev build-essential libssl-dev libreadline-dev libyaml-dev libsqlite3-dev sqlite3 libxml2-dev libxslt1-dev libcurl4-openssl-dev python-software-properties libffi-dev \
```

Comment:
Minor stylistic thing here... can we have these one package per line, in sorted order? Makes merges easier to read.